scripts/reset-project.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+/**
+ * This script is used to reset the project to a blank state.
+ * It moves the /app directory to /app-example and creates a new /app directory with an index.tsx and _layout.tsx file.
+ * You can remove the `reset-project` script from package.json and safely delete this file after running it.
```

Comment:
Looks like you should delete this file instead of committing it

Reply:
idk that was literally there by default teehee